rpm/SPECS/docker-ce.spec

```diff
@@ -16,7 +16,7 @@ Vendor: Docker
 Packager: Docker <support@docker.com>
 
 Requires: docker-ce-cli
-Requires: container-selinux >= 2.9
+Requires: container-selinux >= 2:2.74
```

Comment:
If we're to build this for openSUSE/SLES, we need this:
https://github.com/docker/containerd-packaging/pull/95/commits/072a3aad8bf4c593eceabf3a3f11658792208bdb

Reply:
these specs are only used for Docker CE, there's a spec in the EE packaging repository that does that